### PR TITLE
Commands for: Model, Policy & Request

### PR DESCRIPTION
**Pull request by bjufre**
Hello @Mulkave!

First I'd like to thank you for this beautiful package and for the talk you gave at Laracon, really inspiring. It was because of that talk, and the post you wrote at medium, that I decided to give Lucid a go. So far I f*****g love it :)

But soon I found myself repeating some mechanical actions like creating models by hand or with the artisan cmd, and then moving it, so on and so forth. That's why I wanted to create or better said, implement some commands that would make that process much easier.

Let me know what you think!

And again, thanks!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 composer.lock
 vendor/
+.idea
```
Comment (Mulkave) [inline .gitignore]: Please remove and add to your globally ignored files/directories.

File: src/Finder.php [hunk 2]
```diff
@@ -654,6 +654,92 @@ public function listFeatures($serviceName = '')
...
+    }
+
+    /**
+     * Get the path to the policies directory.
+     *
+     * @return string
+     */
+    public function findPoliciesPath()
+    {
+        return $this->getSourceDirectoryName() . '/Domains/Policies';
```
Comment (Mulkave) [inline src/Finder.php]: I was wondering whether `Domains` is the best place for the `Policies` to exist. Or whether it should have its own directory (i.e. `src/Policies`). Would love to hear your thoughts on that.

**Comment by Mulkave** [comment]
Hello @behind-design!

Awesome work mate! It is with these contributions that such projects become more and more helpful.
I have a few minor questions/comments that I will submit in a few, would love to hear your feedback on them.

Thank you!

**Comment by bjufre** [comment]
@Mulkave Absolutely, waiting to hear them!

File: src/Finder.php (hunk 2 above)
Comment (bjufre) [reply src/Finder.php]: @Mulkave Yeah I totally agree, maybe it's confusing to have them in "Domains" due to its nature. It makes absolute sense! Let's keep them in its own directory: `src/Policies`.

Comment (bjufre) [reply src/Finder.php]: @Mulkave I pushed those changes to the branch.

**Pull request merged**